test/functional/rpc_psbt.py

```diff
@@ -69,6 +69,15 @@ def run_test(self):
         assert_equal(walletprocesspsbt_out['complete'], True)
         self.nodes[1].sendrawtransaction(self.nodes[1].finalizepsbt(walletprocesspsbt_out['psbt'])['hex'])
 
+        # feeRate of 0.1 BTC / KB produces a total fee slightly below -maxtxfee (~0.05280000):
```

Comment:
16257: 
- dashify
- fees are higher in our case - `0.06650000` https://gitlab.com/dashpay/dash/-/jobs/1861193316#L2158
- should also tweak `assert_greater_than`s below accordingly

Reply:
Resolved in b2ca69100c40d1cb9e75aa3fb954ab1e1499a858, thanks a lot!